src/components/energyLabel/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+
+import Text from '../text';
+import Flex from '../flex';
+import Box from '../box';
+
+type Efficiency = 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G';
+
+interface Props {
+  efficiency: Efficiency;
+}
+
+const EnergyLabel: FC<Props> = ({ efficiency }) => {
+  const colors: Record<Efficiency, `#${string}`> = {
```

Comment:
I think it's okay to go 
```suggestion
  const colors: Record<Efficiency, string> = {
```
since it's internal. Up to you.